src/Common/src/CoreLib/System/IO/Path.cs

```diff
@@ -812,8 +812,12 @@ public static string GetRelativePath(string relativeTo, string path)
 
         private static string GetRelativePath(string relativeTo, string path, StringComparison comparisonType)
         {
-            if (string.IsNullOrEmpty(relativeTo)) throw new ArgumentNullException(nameof(relativeTo));
-            if (PathInternal.IsEffectivelyEmpty(path.AsSpan())) throw new ArgumentNullException(nameof(path));
+            if (relativeTo == null)
+                throw new ArgumentNullException(nameof(relativeTo));
+            if (relativeTo.Length == 0)
+                throw new ArgumentException(nameof(relativeTo));
+            if (PathInternal.IsEffectivelyEmpty(path.AsSpan()))
+                throw new ArgumentNullException(nameof(path));
```

Comment:
Please remove this.  This change needs to be made in the dotnet/coreclr repo.  This code is mirrored but isn't actually compiled in corefx.